### DISCUSSION TITLE
How to upgrade LNBits to the latest version.

### DISCUSSION THREAD

**Discussion by marssystems** [raspiblitz/raspiblitz#4252] [Q&A]
How can I update LNBits to the latest version? At the moment there doesn't seem to be any way to do this.

**Comment by rootzoll** [comment]
Normally LNbits get updated if a new RaspiBlitz version is released.

**Comment by marssystems** [comment]
So in the meantime, how do we take advantage of all the new features and updates in the latest version of LNBits.
When can we expect a new release of RaspiBlitz incorporating v11.1 of LNBits?

**Comment by rootzoll** [comment]
We try to ship every new RaspiBlitz version with the latest LNBits.

**Comment by marssystems** [comment]
Yes I know. But in the meantime there should be a script - like for Bitcoin Core, LND and LNDg to update to the latest version from the SSH main menu.
In fact - all apps on RaspiBlitz should be upgradable from the Main Menu in between major releases of RaspiBlitz.

**Comment by rootzoll** [comment]
Shure I would also love to have all the apps to be updatable between releases. But the truth is that those apps sometimes change between versions in how to install land config - for example LNbits just lately introduced a super user or changed databases .. all those need manual adaption and tests of the install scripts that we provide with a new RaspiBlitz update.

For BitcoinCore & LND we provide the RECKLESS updates - but those are just for emergencies or reckless testing. To make this for all apps would too often push the system into unstable states and would multiply the user support work which is already on high workload level.

**Comment by marssystems** [comment]
Well if you can release the next version of RaspiBlitz, incorporating all the latest updates available, in the next week or so, that would be great!

**Comment by MaxSikorski** [comment]
How can we help you on the next release of raspiblitz @rootzoll? What is priority on the list to be completed?

**Reply by rootzoll** [reply]
see https://github.com/raspiblitz/raspiblitz/issues/4402 .. I puhed the update to 0.12.0 to the v1.11.1 release.
